app_grim_reefer/boards/arm/grim_reefer/grim_reefer-pinctrl.dtsi

```diff
@@ -0,0 +1,14 @@
+/ {
+	soc {
+		pinctrl: pin-controller@40020000 {
+            eth_cs_pa8: eth_cs_pa8 {
+                pinmux = <STM32_PINMUX('A', 8, GPIO)>;
+                bias-pull-up;
+            };
+
+            flash_cs_pc0: flash_cs_pc0 {
+                pinmux = <STM32_PINMUX('C', 11, GPIO)>;
+            };
+        };
+    };
+};
```

Comment:
Might not be relevant to the GRIM reefer board

Reply:
fixed in [314d58d](https://github.com/RIT-Launch-Initiative/FSW/pull/52/commits/314d58d64e4811e7a7bd31817aa276b685ae3c5b)